internal/mage/sdk/python.go

```diff
@@ -82,6 +83,11 @@ func (t Python) Generate(ctx context.Context) error {
 	return os.WriteFile(pythonGeneratedAPIPath, []byte(generated), 0600)
 }
 
+// Publish publishes the Python SDK
+func (t Python) Publish(ctx context.Context, tag string) error {
+	return errors.New("not implemented")
```

Comment:
@helderco @gerhard Filling this up is is all we need for `mage sdk:python:publish` (plus creating a `workflows/publish-python.yml`, inspired by the Go one)

Reply:
Do I send a PR based on this one or do you prefer to include Python right away here?

Reply:
By the way, I have a working `mage sdk:python:publish`.

Reply:
https://github.com/dagger/dagger/pull/3730